volume_types.go

```diff
@@ -159,8 +202,11 @@ type VolumeModify struct {
 	//  Size must be a multiple of 8192.
 	Size int64 `json:"size,omitempty"`
 	// Unique identifier of the protection policy assigned to the volume.
-	ProtectionPolicyID  string `json:"protection_policy_id"`
+	ProtectionPolicyID string `json:"protection_policy_id"`
```

Comment:
Will it be always present?
In case if this is not present, will it be okay to add omitempty tag here too?